aiozk/session.py

```diff
@@ -84,6 +84,8 @@ def __init__(self, servers, timeout, retry_policy, allow_read_only):
                 conn = await self.make_connection(host, port)
                 if not conn or (conn.start_read_only and not allow_read_only):
                     continue
+                else:
```

Comment:
@Soniccube yeah, looks like this is broken. Thank you for your pull request.
Could you please just remove `else` statement and put `break` on next line, like:
```
if something:
    continue
break
```
And I will merge it.